src/parser/jobs/whm/index.js

```diff
@@ -13,25 +11,14 @@ This analyser aims to identify some of the low-hanging fruit that could be used
 Note that since healing is a very fight-dependent process, it might be that some suggestions do not apply to you, but they may they help to inform you about things you have missed.
```

Comment:
I realize this is a pre-existing condition, but they might they benefit from cleaning this up while we're here 🙃